docs/tui-guide.md

```
@@ -142,15 +140,16 @@ If the response is interrupted or unsatisfactory, you can regenerate it with `.r
 .info session            Show session info
 .info agent              Show agent info
 .info rag                Show RAG info
+.info tools              List available tools and their active state
 ```
 
-### `.exit` - exit session/agent/RAG/REPL
+### `.exit` - exit the current scope
 
 ```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)